meta-lmp-bsp/recipes-security/optee/optee-os-fio_3.12.0.bbappend

```diff
@@ -1,8 +1,32 @@
+OPTEEMACHINE_apalis-imx6 = "imx-mx6qapalis"
 OPTEEMACHINE_qemuarm64 = "vexpress-qemu_armv8a"
+OPTEEMACHINE_imx6ullevk = "imx-mx6ullevk"
+OPTEEMACHINE_imx8mqevk = "imx-mx8mqevk"
+OPTEEMACHINE_imx8mmevk = "imx-mx8mmevk"
 OPTEEMACHINE_uz = "zynqmp-zcu102"
 
-EXTRA_OEMAKE_append_qemuarm64 = " \
-    CFG_RPMB_FS=y CFG_RPMB_WRITE_KEY=y \
+EXTRA_OEMAKE_append_imx = " \
+    CFG_CAAM_DBG=0x001 \
+    CFG_SCTLR_ALIGNMENT_CHECK=n \
```

Comment:
I think we can now drop CFG_SCTLR_ALIGNMENT_CHECK as it is =n by default (wasn't the case before, iirc at 3.8.0).